frontend/server/src/main/java/com/amazonaws/ml/mms/util/ServerGroups.java

```diff
@@ -45,9 +45,9 @@ public ServerGroups(ConfigManager configManager) {
     public final void init() {
         allChannels = new DefaultChannelGroup(GlobalEventExecutor.INSTANCE);
 
-        serverGroup = NettyUtils.newEventLoopGroup(Runtime.getRuntime().availableProcessors());
+        serverGroup = NettyUtils.newEventLoopGroup(2);
```

Comment:
What is this change for?

Reply:
This is listener event loop thread pool, 1 is enough per listener, we are 2 listener, so it should be 2.